pkg/cloud/aws/actuators/machine/actuator.go

```diff
@@ -619,7 +630,67 @@ func (a *Actuator) UpdateLoadBalancers(client awsclient.Client, providerConfig *
 	}
 
 	if len(errs) > 0 {
-		return fmt.Errorf("failed to register instances with elbs: %v", errs)
+		return fmt.Errorf("failed to register instances with classic ELBs: %v", errs)
+	}
+	return nil
+}
+
+func (a *Actuator) registerWithNetworkLoadBalancers(client awsclient.Client, providerConfig *providerconfigv1.AWSMachineProviderConfig, instance *ec2.Instance) error {
+	glog.V(4).Infof("Updating network load balancer registration for %s\n", *instance.InstanceId)
+	lbNames := []*string{}
```

Comment:
nit: `lbNames := make([]*string, len(providerConfig.LoadBalancerNames)` - as we know up front the length.